### PR TITLE
Adding website

### PR DESCRIPTION
**Pull request by RichardLitt**
This is my first stab at the website. What do you think?

### CODE REVIEW

**Comment by DuaneOBrien** [comment]
Looking at https://deploy-preview-1--fossresponders.netlify.com/ for the preview - are you waiting to add copy before this is ready to merge, or should we aim to merge and add copy in a different PR?

**Comment by RichardLitt** [comment]
@DuaneOBrien Sorry! A couple of interruptions tonight, and longer than I expected to make. Just added in copy. Take a look, and let's edit what we can!

**Comment by RichardLitt** [comment]
Take a look at https://5e6aec4a262a77000a1f70a8--fossresponders.netlify.com!

**Comment by rachellawson** [comment]
I'm nitpicking but the contrast ratio on the main "buttons" is a bit off on hover. It seems that the background changes to white on hover over the button area but the text color only changes to black when you hover over the actual text. It leads to a situation where we have light green text on a white background. I'll see if I can find it in the code...

File: _sass/_custom.scss [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  margin: 0px auto;
+  text-align: center;
+  padding: 20px 40px;
+  margin: 20px;
+  font-size: 20px;
+}
+
+.btn {
+  a, a:visited {
+    color: $secondary-color;
```
Comment (rachellawson) [inline _sass/_custom.scss:102]: I would presume my comment at https://github.com/foss-responders/fossresponders.com/pull/1#issuecomment-598574053 is related to something here. I think the sass ends up setting a specific colour on both the anchor and the containing button div. So, the div:hover only affects the button, leaving the override on the anchor to change only when we reach a:hover

Comment (RichardLitt) [reply _sass/_custom.scss:102]: I don't think this is nitpicking. I should have fixed this at the time, but was blanking as to how. I fixed it using `span`.

**Comment by RichardLitt** [comment]
@rachellawson Fixed that! Any other comments? :)

**Comment by nuritzi** [comment]
@RichardLitt this looks great so far -- thank you so much for getting this up! 

A few questions/thoughts... 
 * **List of buttons**: What's the difference between the first option: "I manage events or communities and need backup" and the last option "We need people to help us organize or respond"? Those seem very similar / the same to me. 

Here's a version of the buttons/list that I think may be more concise:

1. I need help with an upcoming conference that may be affected. 
2. I need financial aid for an event that was cancelled (individual). 
3. We need financial aid for an event that was cancelled (event organizers). 

Does this cover the use cases you were thinking of? 

 * **Funding section**: In the funding section at the bottom of the page, should we remove the first bullet point since we already have a button for people who need financial aid because an event was cancelled? Perhaps I'm missing the difference, but it seems redundant and a bit confusing.

**Comment by RichardLitt** [comment]
@nuritzi Good points! @DuaneOBrien - you wrote those templates. Did you have some Personas in mind that were different, or should we shrink down to three?

**Comment by DuaneOBrien** [comment]
@RichardLitt @nuritzi  the different personas here were:

Community Manager/Conference Organizer who needs emotional support/a partner/intervention (I need some backup)

Events Team/Organization that is scrambling to pull together a virtual event or otherwise meet a surge of activity (We need help pulling this together).

I struggled with getting the right wording between those two, and it shows. I wanted to take Heather's guidance into account when proving support options (Keep an eye on your front line people, they're at risk for burning out the fastest, the best prevention is finding them a partner).

But given the Issue Templates for both are practically identical, that's probably a good signal that we can generalize this just a little and consolidate the two requests into one.

The main thing I really want to be carried away from this is that I want to send some signal that people can and should ask for help if they're going it alone and stressed out. If I was going to propose any rewrite of the "I Need Backup" button it would be

"I'm trying to do this all myself and I need a partner" 

But again - in favor of generalizing and consolidating, especially if it moves us forward quickly.

**Comment by RichardLitt** [comment]
Why not just change the first one to "I'm trying to do this all myself and I need a partner"? The second one works more for organizations that already have support but just need more of it, while the first may require more hand holding. I think they're different, too.

**Comment by DuaneOBrien** [comment]
I'm good with that.

File: index.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+## Sponsors and Backers
+
+FOSS Responders is an open source, open project. Anyone can jump in and work on it. It is entirely volunteer run. Right now, we're in particular need of funders, designers, and people to spread the word.
+
+Some of the companies where our volunteers work (but may or may not be representing their company) include:
+
+- [GitLab](https://about.gitlab.com/)
+- [Google](https://google.com)
+- [Indeed](https://protocol.ai)
```
Comment (DuaneOBrien) [inline index.md]: Should be indeed.com

File: index.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+- [Storj](https://storj.io)
+- ...and yours?
+
+## Other Efforts
+
+While not dedicated entirely to open source, here are some other programs that need help
+
+* [Covid19 Response Fund](https://covid19responsefund.org/). Donate directly to the WHO, who will allocate funds to efforts that need it.
+* [Portland Corona Virus Mutual Aid Open Collective](https://opencollective.com/portland-corona-virus-mutual-aid-fund). Donate to those in Portland who need help.
+
```
Comment (DuaneOBrien) [inline index.md:83]: `* [Organizing Virtual Events Handbook](https://make.wordpress.org/community/handbook/meetup-organizer/resources/organizing-virtual-events/). Assembled by the Wordpress community.`

File: index.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+- Here is [a dynamic list of cancelled events](https://airtable.com/shrETNURgXNrGWbd8/tblc49hMMykARebo8). Not seeing an event? [Add it here](https://airtable.com/shr5QBJUPPOQUJfND).
+- If you were sponsoring a cancelled conference for recruitment purposes and you still want to connect with prospective candidates, Indeed is working on scheduling virtual hiring events. [Click here for more information](https://events.indeed.com/hiringevents/#get_started_form).
+
+What else should be here? [Get in touch](mailto:foss-responders@googlegroups.com!)
+
+
+## Funding
+
+- [Indeed](https://indeed.com) is running another round with $10k to focus on projects impacted by COVID-19. They're looking for others to join in organization, sponsorship, mentorship, and outreach.
```
Comment (DuaneOBrien) [inline index.md]: `[Indeed's Open Source Program Office](https://opensource.indeed.com) and the [Open Collective](https://opencollective.com) are organizing a FOSS Funders event in Q2 with a focus on projects impacted by COVID-19. They're looking for others to join in organization, sponsorship, mentorship, and outreach. Send an email to [opensource@indeed.com](mailto:opensource@indeed.com) for details.`

**Review by DuaneOBrien — COMMENTED** [review]
Clarifying some language around the funding event
Fixes a domain name
Adds the Wordpress Virtual Events Handbook

File: index.md (hunk 2 above)
Comment (RichardLitt) [reply index.md]: No idea how that happened. Will fix.

**Comment by RichardLitt** [comment]
Fixed. @nuritzi How does that sound, to you?

**Comment by megansanicki** [comment]
1st line of copy says: "If you work on open source are affected by Coronavirus, please get in touch. We are actively looking for ways to help out community members."

I recommend that we clarify that we want to help people, communities and industry organizations that were impacted that help can come in many forms, including financial but also advice on how to handle cancellations and how to manage digital community events. So, you may want to expand the copy to say something like 

" COVID-19 is impacting the open source industry in many ways. If you or your community has been impacted, please let us know. We are actively looking for ways to help. Help may come in many forms such as advice on how to cancel/negotiate your event contracts, how to managed digital events, and individual or organizational financial support. "

**Comment by RichardLitt** [comment]
Thanks, @megansanicki. Added.

**Comment by alnermcgunn** [comment]
This paragraph:
"Have something else you need? Please open an issue, join our Slack or email foss-responders@googlegroups.com. If you’d rather email than open a GitHub issue, please feel free to email, as well."

Reads a bit redundantly to me, perhaps change to
"Have something else you need? Please open an issue on GitHub or join our Slack. 
If you’d rather email than open a GitHub issue, please feel free to contact us at foss-responders@googlegroups.com."

Also, are we actually using opencollective slack that is currently linked, or will we set up a dedicated foss-responders slack?

**Comment by RichardLitt** [comment]
Thanks. Fixed.

**Comment by RichardLitt** [comment]
Merged. Let all other comments come in as PRs or as issues.

**Pull request merged**